resources/dicts/patrols/beach/training/any.json

```diff
@@ -1,556 +1,944 @@
+            "platonic",
+            "comfort",
+            "trust",
+            "blunt_force_injury",
+            "death",
+            "scar",
+            "LEFTEAR",
+            "p_l_to_r_c",
+            "s_c_to_r_c",
+            "patrol_to_p_l",
+            "patrol_to_r_c"
+        ],
+        "intro_text": "Trailing behind p_l, the dune {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} crossing shifts beneath r_c's paws. {PRONOUN/r_c/subject/CAP} {VERB/r_c/look/looks} around wildly for safe footing - just as the sand caves in beneath {PRONOUN/r_c/object} and the world goes dark.",
         "decline_text": "Thankfully, they're on the edge of the obstruction, and wriggle their way clear.",
         "chance_of_success": 50,
         "exp": 20,
         "success_text": [
-            "p_l comes running at their yowls, and r_c is soon freed, shaken but unharmed. The cats settle down as they comfort each other.",
+            "p_l comes running at {PRONOUN/r_c/poss} yowls, and r_c is soon freed, shaken but unharmed. The cats settle down as {PRONOUN/r_c/subject} {VERB/r_c/comfort/comforts} each other.",
             "p_l digs through the loose sand, helping r_c clear it away enough for the other cat to scramble free.",
-            "Without letting themselves panic, s_c carefully assesses the collapsed dune trapping their Clanmate. They find stable footing to dig them out from above, and r_c is recovered, scared but okay.",
-            "Creeping in as close as possible, s_c purrs to r_c, keeping them calm as they carefully dig down to them. r_c blinks up at them, holding on to their presence in the scary situation."
+            "Without letting {PRONOUN/r_c/subject} panic, s_c carefully assesses the collapsed dune trapping {PRONOUN/r_c/poss} Clanmate. {PRONOUN/r_c/subject/CAP} {VERB/r_c/find/finds} stable footing to dig {PRONOUN/r_c/object} out from above, and r_c is recovered, scared but okay.",
+            "Creeping in as close as possible, s_c purrs to r_c, keeping {PRONOUN/r_c/object} calm as {PRONOUN/r_c/subject} carefully dig down to {PRONOUN/r_c/object}. r_c blinks up at {PRONOUN/r_c/object}, holding on to {PRONOUN/r_c/poss} presence in the scary situation."
         ],
         "fail_text": [
             "p_l works all day to free r_c and gets nothing else done.",
-            "As s_c rarely has patience for other cats, r_c tells them harshly that they don't have time for some idiot who got stuck, making s_c resentful as they dig themselves out of the sand.",
-            "Hurt and wheezing, r_c hears p_l panicking as they drift to StarClan.",
-            "p_l eventually manages to rescue r_c, but they need to be helped back to camp, battered and injured."
+            "As s_c rarely has patience for other cats, r_c tells {PRONOUN/r_c/object} harshly that {PRONOUN/r_c/subject} don't have time for some idiot who got stuck, making s_c resentful as {PRONOUN/r_c/subject} {VERB/r_c/dig/digs} {PRONOUN/r_c/object} out of the sand.",
```

Comment:
first pronoun tag should be s_c, not r_c, and last 3 pronoun tags should be s_c, not r_c